### PR TITLE
MES-49 Add candidate verification

### PR DESCRIPTION
**Pull request by mathieuhendey**
screen shot 2018-03-22 at 15 17 59

### CODE REVIEW

File: src/pages/declaration-consent/declaration-consent.html [hunk 1]
```diff
@@ -5,22 +5,45 @@
...
+            <br>
+            <span class="mes-text-md">the use of the test vehicle for the purpose of the test is fully covered by a valid policy of insurance which satisfies the requirements of the legislation.</span>
+            <div>
+                <input type="checkbox" name="insurance" id="check-insurance">
+                <label for="check-insurance">I confirm</label>
+            </div>
+            <div class="mes-divider"></div>
+            <span class=" mes-text-md ">I normally live/have lived in the UK for at least 185 days in the last 12 months (except taxi/private hire).</span>
+            <div>
+                <input type="checkbox" name="insurance" id="check-insurance">
```
Comment (ammarhaiderbjss) [inline src/pages/declaration-consent/declaration-consent.html]: Can we have unique `name` and `id` attributes please

File: src/pages/declaration-consent/declaration-consent.html [hunk 2]
```diff
@@ -5,22 +5,45 @@
...
+                <label for="check-insurance">I confirm</label>
```
Comment (ammarhaiderbjss) [inline src/pages/declaration-consent/declaration-consent.html]: This `for` attr will also need to be changed

File: src/pages/declaration-consent/declaration-consent.ts [hunk 3]
```diff
@@ -14,19 +16,56 @@ import { Page } from 'ionic-angular/navigation/nav-util';
...
+    backgroundColor: '#ffffff'
+  };
 
-  constructor(public navCtrl: NavController, public navParams: NavParams) {
+  @HostListener('window:navExpand')
+  onNavExpand(event) {
+    setTimeout(() => {
+      this.signaturePad.resizeCanvas();
+      this.signaturePad.fromData(this.signature);
+    }, 700);
```
Comment (ammarhaiderbjss) [inline src/pages/declaration-consent/declaration-consent.ts]: Can we hold this timeout number in config somewhere? Other components may need to use it too in the future.

File: src/theme/sass-partials/_forms.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.signature-pad {
+  button#retry {
+    float: left;
+  }
+  button#continue {
+    float: right;
+  }
+  background-color: #fafafa;
```
Comment (ammarhaiderbjss) [inline src/theme/sass-partials/_forms.scss]: We can create new colours in _variables.scss which will help us to re-use the same ones and keep track of our overall colour pallette

File: src/pages/declaration-consent/declaration-consent.scss [hunk 5]
```diff
@@ -1,3 +1,8 @@
 page-declaration-consent {
-
+  ion-menu {
+    background-color: #383e3e;
```
Comment (ammarhaiderbjss) [inline src/pages/declaration-consent/declaration-consent.scss]: Can we add to _variables.scss please

File: src/pages/declaration-consent/declaration-consent.ts [hunk 6]
```diff
@@ -14,19 +16,56 @@ import { Page } from 'ionic-angular/navigation/nav-util';
...
+    this.signaturePad.clear(); // invoke functions from szimek/signature_pad API
+  }
+
+  drawComplete() {
+    this.signature = this.signaturePad.toData();
+  }
+
+  clearSignaturePad() {
+    this.signaturePad.clear();
+    console.log('cleared pad');
```
Comment (ammarhaiderbjss) [inline src/pages/declaration-consent/declaration-consent.ts]: We can remove these all console logs

File: src/app/app.component.ts [hunk 7]
```diff
@@ -56,6 +56,8 @@ export class App {
   }
 
   setNavState() {
+    const event = new CustomEvent('navExpand');
+    window.dispatchEvent(event);
     this.navOpen = !this.navOpen;
   }
```
Comment (danielchildsbjss) [inline src/app/app.component.ts]: We might be better off using ionic events - https://ionicframework.com/docs/api/util/Events/

File: src/pages/candidate-info/candidate-info.html [hunk 8]
```diff
@@ -6,77 +6,77 @@
 -->
 
 <ion-header>
-  <page-header></page-header>
+    <page-header></page-header>
```
Comment (zielinskikamil) [inline src/pages/candidate-info/candidate-info.html]: hym, should we agreed on tab vs spaces approach. I'm personally fun of 2 spaces but if everybody loves tabs we could start using tabs. 

Tbh I would like to our code convention be consistent across all our subprojects

Comment (ataylorBJSS) [reply src/pages/candidate-info/candidate-info.html]: @zielinskikamil - the prettier package will help enforce the code styles (tabs, semicolons etc), we just need to evolve it until we are happy with it.

**Pull request merged**